plugins/modules/ec2_vol.py

```diff
@@ -111,6 +111,11 @@
       - This parameter is supported with io1 and io2 volumes only.
     type: bool
     version_added: 2.0.0
+  outpost_arn:
```

Comment:
I guess `version_added` is missing here. If I am not mistaken it should be `3.1.0`.